compiler/rustc_mir_build/src/build/expr/as_rvalue.rs

```diff
@@ -812,20 +814,20 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
 
     // Helper to get a `-1` value of the appropriate type
     fn neg_1_literal(&mut self, span: Span, ty: Ty<'tcx>) -> Operand<'tcx> {
-        let param_ty = ty::ParamEnv::empty().and(ty);
-        let size = self.tcx.layout_of(param_ty).unwrap().size;
-        let literal = Const::from_bits(self.tcx, size.unsigned_int_max(), param_ty);
+        let typing_env = ty::TypingEnv::fully_monomorphized();
```

Comment:
sus but i guess ok

Reply:
well not sus but maybe leave a comment saying this is just a layout of a primitive lol